### ISSUE TITLE
Observing someone opening a safe and unobserving spams the safe window open.

### ISSUE THREAD

**Issue by Xoxeyos** [tgstation/tgstation#42981]
Round ID: 102889

## Reproduction: Observe someone opening the vault safe and unobserve to get the safe window spammed, meaning when you exit out of it another one opens when they go left or right on the dial.

**Comment by Angustmeta** [comment]
Observe has a bunch of shenanigans. This is probably related: 

Examining items in someone's inventory while observing doesn't work. But observe someone and open their backpack, then close observe. A while later if that person puts something in the backpack, it will open for the observer, and the contents of the backpack is now examinable.

**Comment by Xoxeyos** [comment]
Yeah, that also happens, it's pretty annoying to see it pop up in a place where I'm trying to interact with something as a ghost.

**Comment by Xoxeyos** [comment]
You also can't see abilities like what a ling has, or stuff like sec gasmasks. It would be nice to be able to see those.
